fractal_tasks_core/tasks/copy_ome_zarr_hcs_plate.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2022 (C) Friedrich Miescher Institute for Biomedical Research and
+# University of Zurich
+#
+# Original authors:
+# Tommaso Comparin <tommaso.comparin@exact-lab.it>
+# Marco Franzon <marco.franzon@exact-lab.it>
+#
+# This file is part of Fractal and was originally developed by eXact lab S.r.l.
+# <exact-lab.it> under contract with Liberali Lab from the Friedrich Miescher
+# Institute for Biomedical Research and Pelkmans Lab from the University of
+# Zurich.
+"""
+Task that copies the structure of an OME-NGFF zarr array to a new one.
+"""
+import logging
+from typing import Any
+
+import zarr
+from pydantic.decorator import validate_arguments
+
+import fractal_tasks_core
+from fractal_tasks_core.ngff.specs import NgffPlateMeta
+from fractal_tasks_core.ngff.specs import NgffWellMeta
+from fractal_tasks_core.ngff.specs import WellInPlate
+from fractal_tasks_core.tasks.io_models import InitArgsMIP
+from fractal_tasks_core.zarr_utils import open_zarr_group_with_overwrite
+
+logger = logging.getLogger(__name__)
+
+
+__OME_NGFF_VERSION__ = fractal_tasks_core.__OME_NGFF_VERSION__
+
+
+def _get_plate_url_from_image_url(zarr_url: str) -> str:
+    zarr_url = zarr_url.rstrip("/")
+    plate_path = "/".join(zarr_url.split("/")[:-3])
+    return plate_path
+
+
+# def _find_plate_urls_from_zarr_urls(zarr_urls: list[str]) -> list[str]:
```

Comment:
Can we remove this commented function?

Reply:
Done in https://github.com/fractal-analytics-platform/fractal-tasks-core/pull/671/commits/893eb7ced767fbfd31c71a6f7972c5a7c52f4081